modin/pandas/dataframe.py

```diff
@@ -875,29 +878,136 @@ def info(
+            lengths["column"] = max(
+                len(column_label), max(len(pprint_thing(col)) for col in columns)
+            )
+            lengths["dtype"] = len(dtype_label)
+            dtype_spaces = (
+                max(lengths["dtype"], max(len(pprint_thing(dtype)) for dtype in dtypes))
+                - lengths["dtype"]
+            )
+
+            header = put_str(head_label, lengths["head"]) + put_str(
+                column_label, lengths["column"]
             )
-            result = tmp_buf.getvalue()
-            result = result.replace(
-                "pandas.core.frame.DataFrame", "modin.pandas.dataframe.DataFrame"
+            if null_counts:
+                lengths["null"] = max(
+                    len(null_label),
+                    max(len(pprint_thing(x)) for x in non_null_count)
+                    + len(non_null_label),
+                )
+                header += put_str(null_label, lengths["null"])
+            header += put_str(dtype_label, lengths["dtype"], spaces=dtype_spaces)
+
+            output.append(header)
+
+            delimiters = put_str(delimiter * lengths["head"]) + put_str(
+                delimiter * lengths["column"]
             )
-            buf.write(result)
-        return None
+            if null_counts:
+                delimiters += put_str(delimiter * lengths["null"])
+            delimiters += put_str(delimiter * lengths["dtype"], spaces=dtype_spaces)
+            output.append(delimiters)
+
+            return output, lengths
+
+        output.extend([type_line, index_line])
+
+        def verbose_repr(output):
+            columns_line = f"Data columns (total {len(columns)} columns):"
+            header, lengths = get_header()
+            output.extend([columns_line, *header])
+            for i, col in enumerate(columns):
+                i, col, dtype = map(pprint_thing, [i, col, dtypes[col]])
+
+                to_append = put_str(" {}".format(i), lengths["head"]) + put_str(
+                    col, lengths["column"]
+                )
+                if null_counts:
+                    non_null = pprint_thing(non_null_count[col])
+                    to_append += put_str(
+                        "{} non-null".format(non_null), lengths["null"]
+                    )
+                to_append += put_str(dtype, lengths["dtype"], spaces=0)
+                output.append(to_append)
+
+        def non_verbose_repr(output):
+            output.append(columns._summary(name="Columns"))
+
+        if verbose:
+            verbose_repr(output)
+        else:
+            non_verbose_repr(output)
+
+        output.append(dtypes_line)
+
+        if memory_usage:
+            deep = memory_usage == "deep"
+            mem_usage_bytes = self.memory_usage(index=True, deep=deep).sum()
+            mem_line = f"memory usage: {format_size(mem_usage_bytes)}"
+
+            output.append(mem_line)
+
+        buf.write("\n".join(output))
```

Comment:
I think we need one more `"\n"` at the end of the string.

Reply:
done in `bf4ae28`